src/java/org/apache/cassandra/db/streaming/CassandraStreamReceiver.java

```diff
@@ -256,6 +258,15 @@ public void finished()
 
                 // add sstables (this will build non-SSTable-attached secondary indexes too, see CASSANDRA-10130)
                 logger.debug("[Stream #{}] Received {} sstables from {} ({})", session.planId(), readers.size(), session.peer, readers);
+
+                if (cfs.metadata().replicationType().isTracked())
```

Comment:
as mentioned elsewhere, we can't require all tracked streams to use bulk transfer